lib/fastlane/plugin/appcenter/helper/appcenter_helper.rb

```diff
@@ -2,6 +2,19 @@ module Fastlane
   module Helper
     class AppcenterHelper
 
+      # basic utility method to check file types that App Center will accept,
+      # accounting for file types that can and should be zip-compressed
+      # before they are uploaded
+      def self.file_extname_full(path)
```

Comment:
I'm not convinced that we need this function. App Center portal accepts all `.zip` without checking if they really end in `.app.zip` and then verifies their contents, failing if we can't determine the build version & number. So you can't really publish an invalid zip anyway.